src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +45,8 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream().filter(element::equals)
+                .findFirst().orElseThrow();
```

Comment:
```suggestion
        return elements.stream()
             .filter(element::equals)
             .findFirst()
             .orElseThrow();
```
it will be easier to read your code for others